### PR TITLE
PULSE-194

### PR DESCRIPTION
**Pull request by Namanj**
https://phdata.atlassian.net/browse/PULSE-194

@afoerster I'm not exactly sure how to unit test this update? 
I did put this control.sh script on the edge node and ran all three services along with log4j-appender-example, and the log-collector picked up the data from the application.

### CODE REVIEW

**Comment by afoerster** [comment]
@Namanj you aren't actually setting any new values here, just logging. To do this story you'll need to 
- create default/parseable default-alert-engine.yml and default-collection-roller.yml
- if the original is not found, (check with `-f` in control.sh), fall back to those scripts
- make sure the default config files are added to the CWD of the script. 
- create a folder here https://github.com/phdata/pulse/tree/master/cloudera-integration/csd called 'default-configs' and put the configs in that
- modify the CSD makefile so that the default-configs folder is copied into the CSD jar (see how we copy the scripts/images folders in the CSD Makefile

**Comment by afoerster** [comment]
@Namanj merged this mistakenly then reverted it. Please open a new pull request when you push your changes.

**Comment by Namanj** [comment]
Okay got it

File: cloudera-integration/csd/scripts/control.sh [hunk 1]
```diff
@@ -40,6 +40,11 @@ export LOGBACK_CONFIG=${LOGBACK_CONFIG:-$DEFAULT_LOGBACK_CONFIG}
 export KEYTAB_FILE=${KEYTAB_FILE:-"${CONF_DIR}/pulse.keytab"}
 export AKKA_CONF=${AKKA_CONF:-"application.conf"}
 
+export DEFAULT_COLLECTION_ROLLER_CONFIG=$(readlink -f cloudera-integration/csd/default-configs/default-collection-roller.yml)
+export COLLECTION_ROLLER_CONFIG=${COLLECTION_ROLLER_CONFIG:-$DEFAULT_COLLECTION_ROLLER_CONFIG}
```
[the remark this answers is not in the document]
Comment (Namanj) [reply cloudera-integration/csd/scripts/control.sh]: I can check these 2 conditions like so:
if [ -f cloudera-integration/csd/default-configs/default-collection-roller.yml ]; then
    if [ ! -s cloudera-integration/csd/default-configs/default-collection-roller.yml ];then
        action
    fi
fi

But if True, default configs will be empty right?

**Pull request merged**